2025-07-XX-qubes-documentation-migrating-to-read-the-docs.md

```diff
@@ -0,0 +1,54 @@
+
+## How will the migration proceed?
+
+Over the next 20 days, between now and <TODO_YYYY-MM-DD>, the community will test the new documentation. During this period, the current documentation will be frozen, which means that no pull requests will be merged. After the testing period has concluded, the documentation team will evaluate the results and make any changes that are needed. Once the migration is complete, we'll make a final announcement, and the new documentation hosted on Read the Docs will officially replace the current documentation.
+
+## What can I do to help?
+
+Please review the new documentation located at <https://doc.qubes-os.org/en/latest/>. It would be especially helpful to compare the new documentation to the current documentation located at <https://qubes-os.org/doc> and point out any inconsistencies. (When doing this, please focus on accuracy and correctness rather than visual aspects.)
+
+There's a full list of new documentation pages to be reviewed in issue [#8180](https://github.com/QubesOS/qubes-issues/issues/8180). After you've reviewed a page, please leave a comment on that issue to let us know your findings. You can simply copy the entries you reviewed from that list into your comment and add a brief note on what you found problematic, found acceptable, and so on. If you'd like to report a more significant bug or request a feature, you can also [open a separate issue](/doc/issue-tracking/). If you're familiar with reStructuredText and would like to submit a change to the new documentation, please open a pull request against this branch: <https://github.com/QubesOS/qubes-doc/tree/rst>.
+
+If you're looking for a place to start, the following pages were created manually rather than being processed automatically and may merit extra scrutiny:
+
+- https://doc.qubes-os.org/en/latest/introduction/intro.html
+- https://doc.qubes-os.org/en/latest/introduction/video-tours.html
+- https://doc.qubes-os.org/en/latest/developer/general/research.html
+- https://doc.qubes-os.org/en/latest/developer/services/admin-api.html
+
+While we want to know about any problems you spot, please feel free to share any positive feedback you have, as well!
+
+## Acknowledgments
+
+The Qubes OS Project is especially grateful to [Maiska (aka m)](/team/#m) and [Tobias Killer (aka tokidev)](/team/#tobias-killer) for their many hours of work over the course of several years on the conversion of the documentation to Sphinx and reStructuredText. We would also like to thank [unman](/team/#unman) and [Solène Rapenne](/team/#solène-rapenne), in addition to Maiska, for serving the Qubes community as official documentation maintainers.
```

Comment:
The conversion tooling was done by @maiska (with support from @marmarek) @tokidev thoroughly and repeatedly QA'ed the converted and deployed rST documentation. 
I would suggest breaking this down :)